src/owasp-dependency-check-false-positives.xml

```diff
@@ -54,6 +54,13 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty\-tcnative\-classes@.*$</packageUrl>
     <cpe>cpe:/a:netty:netty</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: netty-tcnative-boringssl-static-2.0.52.Final-osx-x86_64.jar
+   ]]></notes>
+    <sha1>87a933c786d4b38355ebbeea684ab48ae88ec265</sha1>
+    <cpe>cpe:/a:chromium_project:chromium</cpe>
```

Comment:
```
2022-08-25T19:56:04.2775687Z [ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.1.0:aggregate (default) on project pulsar: 
2022-08-25T19:56:04.2776396Z [ERROR] 
2022-08-25T19:56:04.2777027Z [ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
2022-08-25T19:56:04.2777476Z [ERROR] 
2022-08-25T19:56:04.2792695Z [ERROR] netty-tcnative-boringssl-static-2.0.52.Final-linux-x86_64.jar: CVE-2011-1797(9.3)
```

There are still several failures. It seems related to os classfier.

Reply:
@tisonkun yes it is. I added the regex instead of the sha to get all the different classifiers